views/layouts/layout.ejs

```diff
@@ -43,19 +43,15 @@
 
     <% /* Auto-injected <link> tags: */ %>
     <!--STYLES-->
-    <link rel="stylesheet" href="/dependencies/bootstrap-4/bootstrap.css">
-    <link rel="stylesheet" href="/dependencies/jquery-ui/jquery-ui.css">
-    <link rel="stylesheet" href="/dependencies/jquery-ui/jquery-ui.structure.css">
-    <link rel="stylesheet" href="/dependencies/jquery-ui/jquery-ui.theme.css">
-    <link rel="stylesheet" href="/styles/importer.css">
+    <link rel="stylesheet" href="/min/production.min.css">
```

Comment:
This was already on the server before I started working on it, know anything about it @chrismcclellan ?